profile/Gemfile

```diff
@@ -0,0 +1,5 @@
+source "https://rubygems.org/"
+
+gem "rake"
+gem "ruby-prof"
```

Comment:
ruby-prof is still on versions 0.x so in practice no difference from what we had, until there is ruby-prof 1.x :)